src/App.css

```diff
@@ -20,3 +20,11 @@
 .App-link {
```

Comment:
Given the changes that you made in the `App` component these original classes are no longer being used (with the exception of the centering in `.App`). I would recommend removing them so that your CSS is easier to manage as the app grows.

This isn't something that necessarily needs to be done in this PR, just wanted to mention it.